editor/editor_spin_slider.cpp

```diff
@@ -36,6 +36,9 @@
 #include "editor_scale.h"
 
 String EditorSpinSlider::get_tooltip(const Point2 &p_pos) const {
+	if (grabber->is_visible()) {
+		return rtos(get_value()) + "\nHold Ctrl to round to integers. Hold Shift for more precise changes.";
```

Comment:
```suggestion
		return rtos(get_value()) + "\n" + TTR("Hold Ctrl to round to integers. Hold Shift for more precise changes.");
```
Makes it translatable.

Reply:
Thanks!